src/olympia/accounts/views.py

```diff
@@ -135,6 +136,7 @@ def register_user(sender, request, identity):
     user = UserProfile.objects.create_user(
         email=identity['email'], username=None, fxa_id=identity['uid'])
     log.info('Created user {} from FxA'.format(user))
+    statsd.incr('accounts.account_created_from_fxa');
```

Comment:
Extra `;` at the end, you've done too much JavaScript :)

Reply:
Hah good catch! 😅